pkg/app/utils.go

```diff
@@ -24,3 +24,10 @@ import (
 func appendRandString(name string) string {
 	return fmt.Sprintf("%s-%s", name, rand.String(5))
 }
+
+// getOpenShiftDBTemplate accepts the application name and returns the
+// db template for that application
+// https://github.com/openshift/origin/tree/master/examples/db-templates
+func getOpenShiftDBTemplate(appName string) string {
+	return fmt.Sprintf("https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/%s-persistent-template.json", appName)
```

Comment:
Can we put the link string as a const? It will be easier that way if we want to update it.